src/less/components/_forms-help.less

```diff
@@ -183,32 +186,26 @@
 // Markup:
 // <div class="form-group form-group__inline">
 //     <label class="control-label">Service Name</label>
-//     <div class="form-readonly-text form-readonly-text__inline">read-only content styling with inline level help </div>
-//     <span class="section-help help-inline help-inline--section help-inline--tooltip section-help--tooltip__inline">
+//     <div class="form-readonly-text">read-only content styling with inline level help </div>
+//     <span class="section-help help-inline help-inline--section help-inline--tooltip">
 //         <span class="hyicon hyicon-helpoff help-icon-inline"></span>
 //         <span class="help-block-inline">I'm a additional Inline Help Text - inlined</span>
 //     </span>
 // </div>
 //
-// Component 5.7
+// Component 5.8
 
 .form-group__inline{
     label{
         display: block;
         margin-bottom: 0;
     }
-    .form-readonly-text__inline{
+    .form-readonly-text{
```

Comment:
Would it be easy enough to just leave the old class in place in the CSS even if the example markup and documentation don't mention it? 

```
.form-readonly-text, .form-readonly-text__inline { }
```

But I'll defer to you guys since you know better who is using what and what the impacts would be. It seems very low-risk to me at the end of the day.